lib/gui/pages/main/controllers/image-selection.js

```diff
@@ -26,18 +26,26 @@ module.exports = function(SupportedFormatsModel, SelectionStateModel, AnalyticsS
    * @type {String[]}
    * @public
    */
-  this.mainSupportedExtensions = _.slice(SupportedFormatsModel.getAllExtensions(), 0, 3);
+  this.mainSupportedExtensions = _.intersection([
```

Comment:
I'm a bit confused by what's going on here. This combines this hard-coded array with the list of _all_ supported extensions, but all of these types are -- as far as I know -- already in that list. If you were going to display them in this order, that would make sense, but right now it sorts them a few lines down, so this doesn't actually do anything special as far as I can tell.